File: src/main/java/com/nirdosh/domain/model/transportation/FlightDetail.java
package com.nirdosh.domain.model.transportation;

import org.joda.time.DateTime;
import org.springframework.format.annotation.DateTimeFormat;

public class FlightDetail {

    @DateTimeFormat(pattern = "dd/MM/yyyy HH:mm")
    public DateTime departureDate;

    @DateTimeFormat(pattern = "dd/MM/yyyy HH:mm")
    public DateTime arrivalDate;

    public FlightType flightType;

    public String from;

    public String to;

    public String airline;

    public String flightNumber;

    public String seatNo;

}

File: src/main/java/com/nirdosh/domain/model/transportation/FlightDetail.scala
package com.nirdosh.domain.model.transportation

import org.joda.time.DateTime

class FlightDetail(
                    var departureDate: DateTime,
                    var arrivalDate: DateTime,
                    var flightType: FlightType,
                    var from: String,
                    var to: String,
                    var airline: String,
                    var flightNumber: String,
                    var seatNo: String
                    ){

}
